Wikipedia/Code/WMFAppViewController.m

```diff
@@ -302,6 +307,18 @@ - (void)appLanguageDidChangeWithNotification:(NSNotification *)note {
     [self configureExploreViewController];
 }
 
+- (void)readingListsSyncProgressDidChange:(NSNotification *)note {
+    NSNumber *progress = note.userInfo[WMFReadingListsController.syncProgressDidChangeFractionCompletedKey];
+
+    // TODO: minimum busy time before showing and a minimum show length
+    UITabBarItem *item = [[self navigationControllerForTab:WMFAppTabTypeSaved] tabBarItem];
```

Comment:
@nambatee  @joewalsh I used the `reading list danger zone` to add 10 lists with 10 articles but `progress` was always `1` here. 

I'm looking at the `Initial progress implementation` PR https://github.com/wikimedia/wikipedia-ios/pull/2077 to get a handle on what's happening with the fraction completed logic, but any pointers are welcome! :)

Reply:
(for future reference: ^ this bug was iOS 10.3 only)